Doc/library/importlib.metadata.rst

```diff
@@ -1,19 +1,19 @@
 .. _using:
 
-=================================
- Using :mod:`!importlib.metadata`
-=================================
```

Comment:
These changes derive from [importlib_metadata](https://github.com/python/importlib_metadata/blob/main/docs/using.rst), so need to be backported there.

Reply:
In python/importlib_metadata#466, I explored backporting these changes, but due to complications they introduced, I've instead decided to abandon support for them in the backport and instead just reference the docs here.